Corona-Warn-App/src/main/res/layout/home_statistics_cards_infections_layout.xml

```diff
@@ -1,7 +1,28 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools">
+    xmlns:tools="http://schemas.android.com/tools"
+    tools:ignore="MissingConstraints">
```

Comment:
is this wanted?

Reply:
I use Flow widget to simplify the layout. Constraints are no longer necessary in this case, but lint check complains about it.